--- CGAL-3.2.1/install_cgal.orig	2006-08-17 15:47:05.000000000 +0200
+++ CGAL-3.2.1/install_cgal	2006-08-17 16:06:47.000000000 +0200
@@ -3161,7 +3161,6 @@
 	        "----------------" \
 	        >> ${_file}
             ${_printf} "%s\n" "#ifndef CGAL_NO_${totest}" >> ${_file}
-	    ${_printf} "%s\n" "#define CGAL_${totest}_VERSION 0" >> ${_file}
 	    ${_printf} "%s\n" "//#define CGAL_USE_${totest} 1" >> ${_file}
             ${_printf} "%s\n\n" "#endif // ! CGAL_NO_${totest}" >> ${_file}
         done
@@ -3221,10 +3220,6 @@
 	else
 	    replace_line "${_file}" "${_tmp2}" "${_tmp3}"
 	fi
-	_tmp2="${totest}_VERSION"
-	_tmp3="`value_of ${_tmp2}`"
-	replace_line "${_file}" "#define CGAL_${_tmp2}" \
-          "#define CGAL_${_tmp2} ${_tmp3:-0}"
     done
 
     cd "${CGAL_DIR}"
